app/models/commit.rb

```diff
@@ -0,0 +1,10 @@
+class Commit < ActiveRecord::Base
+  attr_accessible :commit_oid, :repository
+  attr_accessible :author, :author_date
+  attr_accessible :commit_date, :committer
+
+  has_many :ontology_versions
+  has_many :ontologies, through: :ontology_versions
+
+  belongs_to :repository
+end
```

Comment:
Just a note: You didn't include the commit message into the commit model, which is the only interesting thing a commit has left. However, I don't think we need this information in the database right now.